tests/ui/pages/desktop/collections.py

```diff
@@ -0,0 +1,32 @@
+from pypom import Region
+from selenium.webdriver.common.by import By
+
+from base import Base
+
+
+class Collections(Base):
+    """ Collections page. """
+
+    _name_locator = (By.CSS_SELECTOR, ".item > .info > h3")
+
+    def wait_for_page_to_load(self):
+        self.wait.until(lambda _: self.collections)
+        return self
+
+    @property
+    def collections(self):
+        return [self.Collection(self, el)
+                for el in self.find_elements(*self._name_locator)]
+
+    class Collection(Region):
+        """ Represents an individual collection. """
+
+        _root_locator = (By.CLASS_NAME, 'island')
+
+        @property
+        def is_displayed(self):
+            return self.name.is_displayed()
+
+        @property
+        def name(self):
+            return self.find_element(*self._name_locator)
```

Comment:
You could check that the text is not empty, as the `text` method returns displayed text. A simple way would be to check that it's truthy, which means it will be False for None or an empty string.